mail/mlmmj/Portfile

```diff
@@ -0,0 +1,30 @@
+PortSystem          1.0
+
+name                mlmmj
+version             1.4.6
+categories          mail
+platforms           darwin
+license             MIT
+maintainers         @inucat \
+                    proton.me:inucat
+description         A simple and slim mailing list manager
+long_description    Mlmmj is a simple and slim mailing list manager (MLM) \
+                    inspired by ezmlm. It works with many different Mail \
+                    Transport Agents (MTAs) and is simple for a system \
+                    adminstrator to install, configure and integrate with \
+                    other software. As it uses very few resources, and \
+                    requires no daemons, it is ideal for installation on \
+                    systems where resources are limited, such as Virtual \
+                    Private Servers (VPSes).
+homepage            https://mlmmj.org
+
+master_sites        https://codeberg.org/mlmmj/mlmmj/releases/download/RELEASE_1_4_6
```

Comment:
This should rather be done via `codeberg` PortGroup: https://github.com/macports/macports-ports/blob/master/_resources/port1.0/group/codeberg-1.0.tcl

Reply:
Unfortunately the codeberg portgroup does not have a way to request release downloads so overriding the `master_sites` is still required for that, unless someone wants to improve the codeberg portgroup to add this feature.